src/test/java/org/c4sg/integrationTest/StoryIntegrationTest.java

```diff
@@ -59,9 +47,9 @@ public void createAndGetStory() throws Exception {
         CreateStoryDTO request = buildValidCreateStoryDTO();
         this.mockMvc.perform(
                 post(URI_GET_STORIES, request)
-                .content(asJsonString(request))
-                .contentType(MediaType.APPLICATION_JSON)
-                .accept(MediaType.APPLICATION_JSON_UTF8))
+                        .content(asJsonString(request))
+                        .contentType(MediaType.APPLICATION_JSON)
+                        .accept(MediaType.APPLICATION_JSON_UTF8))
```

Comment:
Please realign the fluent interface according to the previous code, that is, all the dots aligned with the post word.